src/subscription_manager/cli_command/cli.py

```diff
@@ -244,7 +244,7 @@ def main(self, args=None):
         # check for unparsed arguments
         if self.args:
             for arg in self.args:
-                print(_("cannot parse argument: {}").format(arg))
+                print(_("subscription-manager: no such option: {}").format(arg))
```

Comment:
Instead of hardcoding "subscription-manager", i'd use `cmd_name()` from rhsm.utils.

Reply:
Done.